AudibleBoulders/AudibleBoulders
# gitSpyInit file update

#### What's this PR do?
Add spyUpdate executable and pre-commit hook creation to gitSpyInit.js
#### What are the important parts of the code?
gitSpyInit.js
#### How should this be tested by the reviewer?
pull down, run gitSpyInit
run "spyupdate" from command line, should not get any errors
#### Is any other information necessary to understand this?
#### What are the relevant tickets? (Please add `closes`, `refs`, etc)
#### Screenshots (if appropriate)
